core/storage/skill/gae_models.py

```diff
@@ -150,3 +150,30 @@ class SkillSummaryModel(base_models.BaseModel):
     # model was created).
     skill_model_created_on = ndb.DateTimeProperty(required=True, indexed=True)
     version = ndb.IntegerProperty(required=True)
+
+
+class SkillsMasteryModel(base_models.BaseModel):
+    """Mastery model of a user for Oppia Skill.
+
+    This model stores progress of each skill for a given user
+    and returns skill id.
```

Comment:
The model doesn't return anything ... I think you can just say "This model stores the degree of mastery of each skill for a given user." and leave it at that.

Add a note as well on how the ID of the model is constructed (see other model docstrings for details on how to do this).